cmd/okctl/create.go

```diff
@@ -45,6 +45,12 @@ func buildCreateCommand(o *okctl.Okctl) *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "create",
 		Short: "Create commands",
+		Long: `Create various kinds of clusters.
+
+Pre-requisites:
+* okctl create infrastructure-as-code files, and assumes that it is
```

Comment:
Ref my message above + a typo s/create/creates/

Reply:
Changed to `okctl creates various configuration files`.